package.json

```diff
@@ -16,6 +18,7 @@
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-jest": "^23.0.2",
+    "indefinite": "^2.3.1",
```

Comment:
Didn't add `@types/indefinite` due to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40238.